### ISSUE TITLE
Issue with Morag prison

### ISSUE THREAD

**Issue by macce2** [Pyrdacor/Ambermoon-Advanced#22]
I assume this part was meant to work with the following way:

As you enter the prison, you're unable to enter the Cell 
until you kill the Lizard (-only one I guess...!?) and get 
the key.

However, what I found was the lizard (?!) re-appearing
repeatedly, seemingly endlessly (?!) after you have
killed it once; and getting the (same?!) key over and over again.
Anyway, the cell door doesn't seem to work as it should;
no "door lock screen" appears, and you are able to
walk through it (or the wall near it) easily and enter the cell.
The remains there are meant to be important, aren't they ..!?
However, I was unable to figure out what to do there
(no pickable items or anything like that ).. ?!?

**Comment by Pyrdacor** [comment]
Good catch. Basically when you defeat the lizard, it should disable this event. But instead it disables the door event. This is also the reason why you can pass the door then. Will fix soon.
